slices/bash.yaml

```diff
@@ -0,0 +1,10 @@
+package: bash
+
+slices:
+  bins:
+    essential:
```

Comment:
I know `debianutils` seems to only be required by the maintainer scripts, and you don't need those here, but are you sure `base-files` isn't needed?

Reply:
I think bash alone is okay to execute scripts without the `base-files`, but bash script might assume something inside `base-files`.

Reply:
fair enough